python-threatexchange/.devcontainer/zshrc

```diff
@@ -0,0 +1,19 @@
+export SAVEHIST=10000000
+
+# Allow history file to be sooper large
+export HISTFILESIZE=1000000000
+
+# Allow zsh to load as much history as it needs 
+export HISTSIZE=1000000000
+
+# Incrementally add to the history file.
+setopt INC_APPEND_HISTORY
+
+# Make path respect local binaries from pip
+export PATH=$PATH:~/.local/bin
+
+# Add directory containing hmalib to PYTHONPATH. This allows vscode
+# testdiscovery to work
+export PYTHONPATH=$PYTHONPATH:/workspaces/ThreatExchange/hasher-matcher-actioner
```

Comment:
I think this needs to be updated...

Also could you have this also be added to `/home/$unixname/.profile` for those using https://code.visualstudio.com/docs/remote/containers#_personalizing-with-dotfile-repositories

Reply:
Updating the path, but I'm not sure what .profile would do here. Feel free to move the files around. I only got things to a point where they work. I'm _certain_ I have not followed best practices. :)

Reply:
I don't know that the best practice is but the way I use the `.profile` file is in my zshenv:
```
# for machine specific configuration
if [[ -f ~/.profile ]]; then
  source ~/.profile
fi
```
https://github.com/BarrettOlson/dotfiles/blob/master/zshenv#L31-L34